lists.js

```diff
@@ -398,20 +398,18 @@ List.prototype.becomeLinked = function () {
     }
 };
 
-List.prototype.asJSON = function () {
-    function itemToJSON (item) {
-        if (item instanceof List) {
-            return '[' +
-                item.asArray().map(
-                    function (each) {
-                        return itemToJSON(each);
-                    }) +
-                ']';
-        } else {
-            return JSON.stringify(item);
-        }
-    };
-    return itemToJSON(this);
+List.prototype.toJSON = function () {
+    return this.contents;
+};
+
+List.prototype.toCSV = function () {
+    // Expects the list to be properly mappable into a
+    // two-dimensional table.
+    return this.contents.map(function (eachRow) {
+        return eachRow.contents.map(function (eachCell) {
+            return JSON.stringify(eachCell);
```

Comment:
It will! :D 

Snap:
![image](https://cloud.githubusercontent.com/assets/1016697/23252879/abce5ed8-f9b2-11e6-8957-c0b6cf163bd7.png)

LibreOffice:
![image](https://cloud.githubusercontent.com/assets/1016697/23252864/9f73bcb4-f9b2-11e6-9b3d-24209dc76e09.png)